pkgs/tools/misc/unclutter-xfixes/default.nix

```diff
@@ -7,7 +7,7 @@ let version = "1.2"; in
 stdenv.mkDerivation {
   name = "unclutter-xfixes-${version}";
   version = version;
-  
```

Comment:
Another unrelated whitespace-change. In a project the size of nix we need to be careful with those as they easily cause merge conflicts. 

(I'm all :+1: on removing trailing whitespace though!)